### PR TITLE
Update plot dimensions

### PR DESCRIPTION
**Pull request by sjspielman**
This PR addresses some QC report things identified in #469 we'd like to fix.

- Use `theme(aspect.ratio = 1)` instead of `coord_fixed()` for UMAPs
- Bigger width for diagnostic cell type plots
- Ensure heatmaps are at least 4" tall (related to #541)
- Bonus, fix a bug introduced in https://github.com/AlexsLemonade/scpca-nf/pull/574/files
  - We moved some text into an `as_is` chunk with `glue::glue()`, so need to use `{}` for variables, not \``r stuff`\`

### CODE REVIEW

**Comment by jashapiro** [comment]
This all looks good to me. I don't know if you had any plans for further modifications?

Oh, one minor one: if we add `+ xlim(0,1)` to the CellAssign plot, I think it should get the densities to extend to the full range of the plot, would look nice.

**Comment by sjspielman** [comment]
> This all looks good to me. I don't know if you had any plans for further modifications?

No further plans now, just later as part of #541. I'll go ahead and mark this ready for review, and add `xlim()`

**Comment by sjspielman** [comment]
Full range!
Screenshot 2023-11-17 at 2 17 00 PM

File: templates/qc_report/celltypes_supplemental_report.rmd [hunk 1]
```diff
@@ -740,7 +744,8 @@ ggplot(celltype_df) +
     x = "Probability of annotated cell type",
     y = ""
   ) +
-  coord_cartesian(xlim = c(0, 1)) +
+  #coord_cartesian(xlim = c(0, 1)) +
```
Comment (jashapiro) [inline templates/qc_report/celltypes_supplemental_report.rmd:747]: No need to keep the old one around, I don't think
```suggestion
```

**Review by jashapiro — APPROVED** [review]
LGTM!

**Comment by sjspielman** [comment]
After this goes in, I'll regenerate the reports, and move to zipping up example data.

**Pull request merged**